docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -0,0 +1,829 @@
+
+`SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` and `SECRET_SEP24_MORE_INFO_URL_JWT_SECRET` are encryption keys that the Anchor Platform will use to generate short-lived tokens it will add to the URLs provided to the wallet. Your business server must also have these keys in its environment so it can verify the token's signature.
+
+### Test With the Demo Wallet
+
+Wallets should now be able to discover, authenticate, and initiate transactions with your service! Your project and source files should now look something like this.
+
+<CodeExample>
+
+```
+├── dev.env
+├── docker-compose.yaml
+├── config
+│   ├── dev.config.yaml
+│   ├── dev.stellar.toml
+```
+
+</CodeExample>
+
+Your environment should now look like the following.
+
+<CodeExample>
+
+```bash
+# dev.env
+ANCHOR_CONFIG_FILE=/home/dev.config.yaml
+
+SEP1_ENABLED=true
+SEP1_TYPE=file
+SEP1_VALUE=/home/dev.stellar.toml
+
+SEP10_ENABLED=true
+SECRET_SEP10_SIGNING_SEED="a Stellar private key"
+SECRET_SEP10_JWT_SECRET="a secret encryption key"
+
+SEP24_ENABLED=true
+SEP24_INTERACTIVE_URL_BASE_URL=http://localhost:8081
+SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET="your encryption key shared with your business server"
+SECRET_SEP24_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your business server"
+```
+
+</CodeExample>
+
+To test this out, go to the [Stellar Demo Wallet][stellar-demo-wallet].
+
+![demo wallet connected to the anchor platform](/assets/anchor-platform-sep24-demo-wallet.png)
+
+Initiate a transaction by doing the following:
+
+- Create a new keypair
+- Click the "Add Asset" button and enter
+  - the code of the Stellar asset on your `stellar.toml` file
+  - your home domain, `localhost:8080`
+- Select the dropdown and click "SEP-24 Deposit", then click "Start"
+
+The demo wallet should be able to find your `stellar.toml` file, authenticate using the Stellar keypair you just created, and initiate a transaction. However, when the demo wallet attempts to open the URL provided by the Anchor Platform, you'll get a not found page.
+
+![demo wallet after initiating a transaction](/assets/anchor-platform-sep24-demo-wallet-widget.png)
+
+### Add Data Persistence
+
+Before we move forward, let's add a database to our development environment so the transactions we initiate persist after stopping the service.
+
+<CodeExample>
+
+```yaml
+# docker-compose.yml
+version: "3.8"
+services:
+  platform-server:
+    image: stellar/anchor-platform:edge
+    command: --sep-server
+    env_file:
+      - ./dev.env
+    volumes:
+      - ./config:/home
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+  db:
+    image: postgres:latest
+    ports:
+      - "5432:5432"
+    env_file:
+      - ./dev.env
+```
+
+</CodeExample>
+
+Now let's update our environment so the platform server can connect to the database server.
+
+<CodeExample>
+
+```bash
+# dev.env
+DATA_TYPE=postgres
+DATA_URL=jdbc:postgresql://db/platform
```

Comment:
Should it be updated according to
https://github.com/stellar/java-stellar-anchor-sdk/pull/759 ?

Reply:
Yes, now that it's updated.